packages/core/hooks/useKeyDown/useKeyDown.ts

```diff
@@ -0,0 +1,23 @@
+import * as React from "react";
+
+export function useKeyDown(targetKeys: string[], handler: (event: KeyboardEvent) => void) {
```

Comment:
When there is only one export, making it the default is generally best
```suggestion
export default function useKeyDown(targetKeys: string[], handler: (event: KeyboardEvent) => void) {
```

Reply:
resolved in [700014c](https://github.com/AllenInstitute/biofile-finder/pull/223/commits/700014c75843ccf83e48c6f8b6ba1b32ad47c55b)